pysat/_orbits.py

```diff
@@ -2,6 +2,10 @@
 # Full license can be found in License.md
 # Full author list can be found in .zenodo.json file
 # DOI:10.5281/zenodo.1199703
+#
+# DISTRIBUTION STATEMENT A: Approved for public release. Distribution is
```

Comment:
Strictly speaking the pysat license determines the limits of distribution, not NRL. Once uploaded to a pysat repo code is covered by pysat's license and is no longer under NRL's control.

pysat also predates any NRL involvement so NRL statements 
 - shouldn't go in every file 
 - should acknowledge that NRL is only a partial contributor and not a controlling institution

A more complete list of funding institutions is in #1168